cli/cmd/upgrade.go

```diff
@@ -93,42 +109,37 @@ and apply any required migrations and adaptations.`,
 				}
 			}
 		} else {
-			operation = "Focefully upgrading"
-		}
-
-		config, err := resources.GetCurrentConfig(ctx, client, ns)
-		if err != nil {
-			fmt.Println("Odigos upgrade failed - unable to read the current Odigos configuration.")
-			os.Exit(1)
-		}
-
-		// update the config on upgrade
-		config.ConfigVersion += 1
-		if uiMode != "" {
-			config.UiMode = common.UiMode(uiMode)
+			operation = "Forcefully upgrading"
 		}
 
-		// Migrate images from prior to registry.odigos.io
-		if config.ImagePrefix == "" {
-			config.ImagePrefix = k8sconsts.OdigosImagePrefix
-		}
+		onPremTokenSet := cmd.Flag("onprem-token").Changed
+		var currentTier common.OdigosTier
 
-		currentTier, err := odigospro.GetCurrentOdigosTier(ctx, client, ns)
-		if err != nil {
-			fmt.Println("Odigos cloud login failed - unable to read the current Odigos tier.")
-			os.Exit(1)
+		if onPremTokenSet {
+			currentTier = common.OnPremOdigosTier
+		} else {
+			currentTier, err = odigospro.GetCurrentOdigosTier(ctx, client, ns)
+			if err != nil {
+				fmt.Println("Odigos cloud login failed - unable to read the current Odigos tier.")
```

Comment:
I think it's an existing copy-paste mistake 😅.

this is not `odigos cloud login` command so we can update the text while at it :)